Emby.Server.Implementations/ApplicationHost.cs

```diff
@@ -1201,14 +1201,12 @@ public string GetSmartApiUrl(string hostname, int? port = null)
         }
 
         /// <inheritdoc/>
-        public string GetLoopbackHttpApiUrl()
+        public string GetUrlForUseByHttpApi()
         {
-            if (NetManager.IsIP6Enabled)
-            {
-                return GetLocalApiUrl("::1", Uri.UriSchemeHttp, HttpPort);
-            }
+            var bind = NetManager.GetInternalBindAddresses().FirstOrDefault() ??
+                NetManager.GetAllBindInterfaces(true).FirstOrDefault();
 
-            return GetLocalApiUrl("127.0.0.1", Uri.UriSchemeHttp, HttpPort);
+            return GetLocalApiUrl(bind.Address.ToString(), Uri.UriSchemeHttp);
```

Comment:
You're using `FirstOrDefault` but you're not handling the null here (I assume that's the default)

Reply:
Changed to First() - as an address will always be returned.